mongodbatlas/access_tracking.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const accessTrackingBasePath = "api/atlas/v1.0/groups/%s/dbAccessHistory"
+
+// AccessTrackingService is an interface for interfacing with the Access Tracking endpoints of the MongoDB Atlas API.
+//
+// See more: https://docs.atlas.mongodb.com/reference/api/access-tracking/
+type AccessTrackingService interface {
+	ListByCluster(context.Context, string, string, *AccessLogOptions) (*AccessLogSettings, *Response, error)
+	ListByHostname(context.Context, string, string, *AccessLogOptions) (*AccessLogSettings, *Response, error)
+}
+
+// AccessTrackingServiceOp handles communication with the AccessTrackingService related methods of the
+// MongoDB Atlas API.
+type AccessTrackingServiceOp service
+
+var _ AccessTrackingService = &AccessTrackingServiceOp{}
+
+// AccessLogOptions represents the query options of AccessTrackingService.List.
+type AccessLogOptions struct {
+	Start      string `url:"start,omitempty"`      // Timestamp in the number of milliseconds that have elapsed since the UNIX epoch for the first entry that Atlas returns from the database access logs.
```

Comment:
[nit] good on documenting the properties but can we follow godoc styles? `// Start is the